CHANGELOG.md

```diff
@@ -41,8 +41,10 @@ The format is based on the [KeepAChangeLog] project.
 
 ### Fixed
 - [#515]: Fix arguments to WSGI start_response
+- [#528]: Fix faulty redirect_uri with query
```

Comment:
This needs to be in the `[Unreleased]` section as it is going to be in the next release.